docs/CNIGenieFeatureSet.md

```diff
@@ -8,22 +8,25 @@
 **Feature 2: CNI-Genie "Multiple IP Addresses"**
 * Injects multiple IPs to a single container. The container is reachable using any of the [multiple IP Addresses](multiple-ips/README.md)
 
-**Feature 3: CNI-Genie "Smart CNI Plugin Selection"**
+**Feature 3: CNI-Genie "Network Attachment Definition"**
```

Comment:
Please update this in "CNI-Genie/README.md" page as well

Reply:
I have updated the document accordingly.